user-guide/Advanced_Functionality/Databases/Specifying_TTL_overrides.md

```diff
@@ -9,7 +9,9 @@ The "time to live" (TTL) determines how long specific data is retained before be
 This page explains how to configure TTL settings to optimize your database performance and resource utilization.
 
 > [!NOTE]
-> If a DMS uses [Storage as a Service](xref:STaaS), the default TTL is automatically applied for most database records. The configuration in Cube is only taken into account for real-time trending.
+>
+> - If a DMS uses [Storage as a Service](xref:STaaS), the default TTL is automatically applied for most database records. The configuration in Cube is only taken into account for real-time trending.
+> - From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39173-->, the TLL settings on the *System settings* > *time to live* tab in System Center are no longer available if you are using a [DaaS system](xref:Creating_a_DMS_in_the_cloud), as these are automatically configured as part of DaaS.
```

Comment:
I think here too we should rephrase the note to highlight the fact that this is automatically configured as part of DaaS, with the fact that it's no longer available now as secondary information.

Reply:
Indeed, this is managed by 'us' (SLC) and irrelevant for the user.